react/components/EXPERIMENTAL_Table/DataTable/Cell.tsx

```diff
@@ -48,22 +55,49 @@ const Cell: FC<CellProps> & CellComposites = ({
       z1: !sticky,
     }
   )
-  const Tag = header ? 'th' : 'td'
+  const tag = header ? 'th' : 'td'
+  const style = {
+    position: sticky ? 'sticky' : 'static',
+    width,
+  } as CSSProperties
+  const Container = sortable ? HoverableCell : DefaultCell
 
   return (
-    <Tag
-      {...events}
-      onClick={onClick}
-      style={{
-        position: sticky ? 'sticky' : 'static',
-        width,
-      }}
-      className={className}>
+    <Container tag={tag} onClick={onClick} style={style} className={className}>
+      {children}
+    </Container>
+  )
+}
+
+interface CellContainer
+  extends React.DetailedHTMLProps<
+    React.ThHTMLAttributes<HTMLTableHeaderCellElement>,
+    HTMLTableHeaderCellElement
```

Comment:
```suggestion
  extends DetailedHTMLProps<
   ThHTMLAttributes<HTMLTableHeaderCellElement>,
    HTMLTableHeaderCellElement
```
and import them from react.

Reply:
okay, nice catch